config/variants/enterprise-postgres/kustomization.yaml

```diff
@@ -6,4 +6,3 @@ patchesStrategicMerge:
 images:
 - name: kong
   newName: kong/kong-gateway
-  newTag: 2.4.1.0-alpine
```

Comment:
Copying from chat discussion earlier, these should continue to have their own tag declarations, even when the version here does match the version in `config/base/kong-ingress-dbless.yaml`, e.g.

```
images:
- name: kong
  newName: kong/kong-gateway
  newTag: "2.5"
```
The reason we have the image changes in these Enterprise kustomizations is that Enterprise manifests use the `kong/kong-gateway` image, not the `kong` image [used in the base Deployment](https://github.com/Kong/kubernetes-ingress-controller/blob/218be9b4b9847d626055ac2e47f02921b0ff9d56/config/base/kong-ingress-dbless.yaml#L25).

Although the `kong/kong-gateway` versions are often the same as the `kong` versions, `kong/kong-gateway` is a separate image with its own release cycle. We can and do release new `kong` tags before an equivalent `kong/kong-gateway` tag is made available, so we always want the Enterprise kustomizations to include the specific image version we want.

We don't want to have to re-add the `newTag` override in the event that we need the `kong` and `kong/kong-gateway` to differ and then take it away again if they happen to be the same.

Reply:
Add newTag back 221ac9580a43ce3f47413ea51688a90bd6799440